modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -25,34 +25,60 @@ static IP address (`ip=<host_ip>`). Then identify the DNS server IP address (`na
 * The hostname to `core0.example.com` +
 * The DNS server address to `4.4.4.41`
 
-|`ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none nameserver=4.4.4.41` +
+a|
+----
+ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none
+nameserver=4.4.4.41
+----
 
 |Specify multiple network interfaces by specifying multiple `ip=` entries.
-|`ip=192.168.122.25 ip=192.168.122.26`
+a|
+----
+ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none
+ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp2s0:none
+----
 
 |You can combine DHCP
 and static IP configurations on systems with
 multiple network interfaces.
-|`ip=enp1s0:dhcp ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp2s0:none`.
+a|
+----
+ip=enp1s0:dhcp
+ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp2s0:none
+----
 
-|You can provide multiple DNS servers by adding a `nameserver=` entry for each server
-|`nameserver=1.1.1.1 nameserver=8.8.8.8`.
+|You can provide multiple DNS servers by adding a `nameserver=` entry for each server.
+a|
+----
+nameserver=1.1.1.1
+nameserver=8.8.8.8
+----
 
 a|Bonding multiple network interfaces to a single interface is optionally supported
 using the `bond=` option.  In these two examples:
 
-* _name_ is the bonding device name (bond0), _network_interfaces_
+* The syntax for configuring a bonded interface is: `bond=name[:network_interfaces][:options]`
+* _name_ is the bonding device name (`bond0`), _network_interfaces_
 represents a comma-separated list of physical (ethernet) interfaces (`em1,em2`),
-and _options_ is a comma-separated list of bonding options.
+and _options_ is a comma-separated list of bonding options. (Type `modinfo bonding` to see available options.)
```

Comment:
s/Type/Enter/ ?
I've been recommending "enter" depending on how it is used elsewhere in the module/assembly/book. Quick scan of this book didn't turn up results either way, so this seems fine as you have it.

Reply:
Oh, yeah. I've heard that before. Okay.

Reply:
Fixed.